server/command/command.go

```diff
@@ -1062,6 +1068,51 @@ func (r *Runner) actionTest(args []string) {
 	}
 }
 
+func (r *Runner) actionTestGeneratePlaybooks(params []string) {
+	if len(params) < 1 {
+		r.postCommandResponse("The command expects one parameter: <numPlaybooks>")
+		return
+	}
+
+	numPlaybooks, err := strconv.Atoi(params[0])
+	if err != nil {
+		r.postCommandResponse("Error parsing the first argument. Must be a number.")
+		return
+	}
+
+	if numPlaybooks > 5 {
+		r.postCommandResponse("Maximum number of playbooks is 5")
+		return
+	}
+
+	playbookIds := make([]string, 0, numPlaybooks)
+	for i := 0; i < numPlaybooks; i++ {
+		dummyPlaybook := dummyListPlaybooks[i]
+		dummyPlaybook.TeamID = r.args.TeamId
+		newPlaybookID, err := r.playbookService.Create(dummyPlaybook, r.args.UserId)
+		if err != nil {
+			r.warnUserAndLogErrorf("unable to create playbook: %v", err)
+			return
+		}
+
+		playbookIds = append(playbookIds, newPlaybookID)
+	}
+
+	team, err := r.pluginAPI.Team.Get(r.args.TeamId)
+	if err != nil {
+		r.warnUserAndLogErrorf("unable to retrieve team: %v", err)
+		return
+	}
+
+	msg := "Playbooks successfully created"
+	for i, playbookID := range playbookIds {
+		url := fmt.Sprintf("/%s/com.mattermost.plugin-incident-management/playbooks/%s", team.Name, playbookID)
+		msg += fmt.Sprintf("\n- [%s](%s)", dummyListPlaybooks[i].Title, url)
+	}
+
+	r.postCommandResponse(fmt.Sprintf("Playbooks successfully created : '%s'", msg))
```

Comment:
We can just use `msg` here. Otherwise, the words "Playbooks successfully created" will be repeated:

```suggestion
	r.postCommandResponse(msg)
```